db/dcrpg/upgrades.go

```diff
@@ -623,6 +661,29 @@ func (pgb *ChainDB) handleUpgrades(client *rpcutils.BlockGate,
 		if err = pgb.ReindexAddressesBlockTime(); err != nil {
 			return false, fmt.Errorf("failed to reindex addresses table: %v", err)
 		}
+
+	case agendasTableReorgUpdate:
```

Comment:
"`Reorg`" is to suggestive of a chain reorg.  Perhaps something more along the lines of "agendasTableV2" or "agendasTablePruning".